tests/platform_tests/daemon/test_pcied.py

```diff
@@ -154,26 +121,27 @@ def test_pmon_pcied_term_and_start_status(check_daemon_status, duthosts, rand_on
     @summary: This test case is to check the pcied terminated and restarted status
     """
     duthost = duthosts[rand_one_dut_hostname]
+
+    if "201811" in duthost.os_version or "201911" in duthost.os_version:
```

Comment:
SIGTERM signal is standard signal and the behavior should be same for all POSIX compliant systems. Can you elaborate what different behavior are you seeing across different os version?

Reply:
@prgeor   please see the previous comment.